README.md

```diff
@@ -42,38 +42,51 @@ Then generate the include file with
 php artisan vue-i18n:generate
 ```
 
-Assuming you are using a recent version of vue-i18n, adjust your vue app with something like:
+Assuming you are using a recent version of vue-i18n (> 6.x), adjust your vue app with something like:
 ```js
 import Vue from 'vue';
 import VueInternalization from 'vue-i18n';
-import Locales from './vue-i18n-locales.generated.js';
+import Locale from './vue-i18n-locales.generated';
 
 Vue.use(VueInternalization);
 
-Vue.config.lang = 'en';
+const lang = document.documentElement.lang.substr(0, 2); 
+// or however you determine your current app locale
 
-Object.keys(Locales).forEach(function (lang) {
-  Vue.locale(lang, Locales[lang])
+const i18n = new VueInternalization({
+    locale: lang,
+    messages: Locale
 });
 
-...
+const app = new Vue({
+    el: '#app',
+    i18n,
+    components: {
+       ...
+    }
+}
 ```
 
-
 For older vue-i18n, the initialization looks something like:
-
 ```js
 import Vue from 'vue';
-import VueInternationalization from 'vue-i18n';
+import VueInternalization from 'vue-i18n';
```

Comment:
This is a typo. i18n = Internationalization,  not Internalization.
https://en.wikipedia.org/wiki/Internationalization_and_localization#Naming

Reply:
Well, I have copied it from the current readme. Did not notice. 

Fixed anyway! :+1:

Reply:
Thanks, I've missed that, should fix!